helm/templates/standalone/statefulset-standalone.yaml

```diff
@@ -92,6 +106,10 @@ spec:
               subPath: {{ .Values.persistence.subPath }}
             - name: config
               mountPath: /etc/mongodb
+            {{- if .Values.usePassword }}
```

Comment:
Can the init-scripts flag be named accordingly (instead of using the `usePassword` flag for `initi-scripts`?